app/tasks/maintenance/t20250226_fix_dossiers_en_instruction_with_pending_correction_task.rb

```diff
@@ -22,6 +22,14 @@ def process(dossier)
         correction = dossier.pending_correction
         dossier.resolve_pending_correction
 
+        if dossier.procedure.sva_svr_enabled?
+          class << dossier.procedure
+            def sva_svr_enabled?
+              false # contourne guard can_repasser_en_construction?
+            end
+          end
```

Comment:
Je connaissais pas cette syntax, p-e du pinaillage, mais se serait pas plus clair ainsi ? 

```suggestion
          dossier.procedure.define_singleton_method(:sva_svr_enabled?) do
            false # contourne guard can_repasser_en_construction?
          end
```